### ISSUE TITLE
Lib: Do not use H1 as the default header level in components.

### ISSUE THREAD

**Issue by bcullman** [SAP/fundamental#1178]
looking at https://sap.github.io/fundamental/components/, some components have a `<H1>` tag, 

* ActionBar
* MegaMenu
* Menu
* Modal
* Panel
* SideNavigation

From an accessibility perspective, a page should have exactly one `<H1>` tag on the page. Having all your examples use `<H1>` sets a bad precedent.

It would be preferable to use `<H3>` as a default in your examples, and explain in the text that `H1` to `H6` is supported
